d2/src/main/java/com/linkedin/d2/balancer/properties/UriPropertiesMerger.java

```diff
@@ -23,9 +23,14 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 
 public class UriPropertiesMerger implements ZooKeeperPropertyMerger<UriProperties>
 {
+  private static final Logger _log = LoggerFactory.getLogger(UriPropertiesMerger.class);
```

Comment:
I was going to comment that I thought it's convention to use `LOG` for a `static final` logger, but I see that the [existing codebase](https://github.com/search?q=repo%3Alinkedin%2Frest.li%20%22private%20static%20final%20Logger%22&type=code) is inconsistent 😂 So I think this is fine

Reply:
yeah, makes sense, just fixed!